routes/html/index.js

```diff
@@ -1,20 +1,27 @@
 const Users = require("../../models/Users");
 const Games = require("../../models/Games");
 const sequelize = require("../../config/connection");
+const { withAuth } = require("../../utils/auth");
 
 const router = require("express").Router();
 
-router.get("/", (req, res) => {
+router.get("/", withAuth, async (req, res) => {
   const loggedIn = req.user;
-  if (loggedIn) {
+  if (!!loggedIn) {
     // render the dashboard
-    res.render("dashboard", { loggedIn });
+    const fetchName = req.user.firstName;
```

Comment:
if the variable is logged in (implies its boolean), then how are you able to get some values from it?